packages/api/src/modules/admin-structure/repositories/adminStructure.repository.ts

```diff
@@ -0,0 +1,32 @@
+import { AgentTypeEnum } from "@api-subventions-asso/dto";
+import { WithId } from "mongodb";
+import db from "../../../shared/MongoConnection";
+import AdminStructureEntity from "../entities/AdminStructureEntity";
+
+export class AdminStructureRepository {
+    private readonly collection = db.collection<AdminStructureEntity>("admin-structures");
+
+    private toEntity(document: WithId<AdminStructureEntity>) {
+        return new AdminStructureEntity(document.agentType, document.territoryScope, document.structure);
```

Comment:
Devrais etre le job d'un adatpeur dbo to entity mais c'est déjà mieux que ce qu'on fait partout alors tu peut laisser